webknossos/pyproject.toml

```diff
@@ -31,6 +31,7 @@ wkw = "1.1.11"
 [tool.poetry.dev-dependencies]
 # autoflake
 black = "21.7b0"
+hypothesis = "^6.35.0"
```

Comment:
@normanrz Using an MPL2 licensed lib as a dev-dependency should be fine, right?

Reply:
Yep